src/vs/code/browser/workbench/workbench.ts

```diff
@@ -91,6 +91,8 @@ class SecretStorageProvider implements ISecretStorageProvider {
 
 	config.secretStorageProvider = new SecretStorageProvider();
 
+	config.commands = [{ id: 'membrane.refreshPage', handler: () => window.location.reload() }];
```

Comment:
Is this the right place for this registration? I think it would likely be better to be placed in the extension. I see that it being here would mean it only shows up on browser installs, but I'm not sure that's really relevant anymore.

Reply:
I think it's the right place because this command requires `window`, which is not available for the extension in a Web Worker (similar to my [other comment](https://github.com/membrane-io/mnode/pull/169#discussion_r1664639303)). I'll double check this as well.

Reply:
Yep, same `ReferenceError: window is not defined`